plugins/scaffolder/src/api.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ConfigApi } from '@backstage/core';
+import { ScaffolderClient } from './api';
+
+describe('api', () => {
+  const discoveryApi = {} as any;
+  const identityApi = {} as any;
+  const configApi = {
+    getOptionalConfigArray: (key: string) => {
+      const config = {
+        'integrations.github': [
+          {
+            getOptionalConfigArray: () => [],
+            getOptionalString: (intKey: string) => {
+              const props = {
+                host: 'hello.com',
+              };
+              return (props as any)[intKey];
+            },
+          },
+        ],
+      };
+      return (config as any)[key];
+    },
+  } as ConfigApi;
+  const apiClient = new ScaffolderClient({
+    configApi,
```

Comment:
you can actually do

```
new ConfigReader({
  integrations: {
    github: [{
      host: ...
```

:)

Reply:
Yay, I was looking for how to instantiate a Config obj 😃